packages/http-server/src/server.ts

```diff
@@ -49,13 +49,86 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
 
   const prism = createInstance(mergedConfig, components);
 
+  const replyHandler: fastify.RequestHandler<IncomingMessage, ServerResponse> = async (request, reply) => {
+    const {
+      req: { method, url },
+      body,
+      headers,
+      query,
+    } = request;
+
+    const input = {
+      method: (method ? method.toLowerCase() : 'get') as HttpMethod,
+      url: {
+        path: (url || '/').split('?')[0],
+        query,
+        baseUrl: query.__server,
+      },
+      headers,
+      body,
+    };
+
+    request.log.info({ input }, 'Request received');
+    try {
+      const operationSpecificConfig = getHttpConfigFromRequest(input);
+      const mockConfig = Object.assign({}, opts.config.mock, operationSpecificConfig);
```

Comment:
How about:
```suggestion
      const mockConfig = { ...opts.config.mock, ...operationSpecificConfig };
```
or
```suggestion
      const mockConfig = _.defaults(opts.config.mock, operationSpecificConfig);
```
?

`Object.assign` is soo ECMA 2015 :/

Reply:
Ahaha, you can definitely see I'm 30 :trollface: